razorpay.php

```diff
@@ -38,6 +38,25 @@ public static function load()
         GFAddOn::register('GFRazorpay');
 
         add_filter('gform_currencies', function (array $currencies) {
+
+            $supported_currencies = json_decode(file_get_contents(__DIR__ . "/supported-currencies.json"), true)['supported-currencies'];
+
+            foreach ($supported_currencies as $k => $v)
+            {
+                if (in_array($v['iso_code'], $currencies) === false)
```

Comment:
is the condition required here key in_array required here

Reply:
@abdulwahidsharief I have put the condition so that any pre defined currencies may not be overwritten.